pkg/machines/scripts/create_machine.sh

```diff
@@ -12,16 +12,15 @@ START_VM="$8"
 STORAGE_POOL="$9"
 STORAGE_VOLUME="${10}"
 UNATTENDED="${11}"
-USER_PASSWORD="${12}"
-ROOT_PASSWORD="${13}"
-PROFILE="${14}"
+ROOT_PASSWORD="${12}"
+PROFILE="${13}"
 
 vmExists(){
    virsh -c "$CONNECTION_URI" list --all | awk  '{print $2}' | grep -q --line-regexp --fixed-strings "$1"
 }
 
 err_handler () {
-    rm -f "$ROOT_PASSWORD_FILE $USER_PASSWORD_FILE $XMLS_FILE"
+    rm -f "$ROOT_PASSWORD_FILE $XMLS_FILE"
```

Comment:
The quoting here is funny.  Doesn't each argument need to be quoted separately, like
```
rm -f "$ROOT_PASSWORD_FILE" "$XMLS_FILE"
```
Did this ever work?

Reply:
You are right, I guess it never worked, and we left the password files lying around :/ I will fix